src/js/utils/functions.js

```diff
@@ -0,0 +1,8 @@
+export function pick(o, fields) {
```

Comment:
I don't why I didn't do this.. lodash is publishing each of its functions separately https://www.npmjs.com/package/lodash.pick